torchbenchmark/models/BERT_pytorch/bert_pytorch/model/attention/single.py

```diff
@@ -16,7 +16,11 @@ def forward(self, query, key, value, dropout: TensorToTensor, mask: Optional[tor
                  / math.sqrt(query.size(-1))
 
         if mask is not None:
-            scores = scores.masked_fill(mask == 0, -1e9)
+            if scores.dtype == torch.float16:
+                min_mask = -65504.0
```

Comment:
Is there any constant defined like 'torch.float16.MIN_VALUE' or 'torch.float.MIN_VALUE' so that we don't need to hardcode the value?